### PR TITLE
Feature/cicd pipeline0508

### LINKED ISSUE #2: Setup Heroku Deployment
Setup Heroku Deployment

### LINKED ISSUE #3: Add Docker Compose
Add Docker Compose

### PR DESCRIPTION
**Pull request by ranjanr2**
Initial Build to Test the Docker file and Heroku Deploy

### CODE REVIEW

**Review by zxander-D — APPROVED** [review]
LGTM

**Review by 404-bug-not-found — CHANGES_REQUESTED** [review]
Need Docker file for heroku deployment

**Comment by ranjanr2** [comment]
Sunita just pushed her changes. please check

**Review by zxander-D — APPROVED** [review]

File: src/main/java/com/hundred/percent/capstone/Invoicify/company/DatabaseConfig.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.hundred.percent.capstone.Invoicify.company;
```
Comment (404-bug-not-found) [inline src/main/java/com/hundred/percent/capstone/Invoicify/company/DatabaseConfig.java:1]: DatabaseConfig is generic class for whole service, So, please move this out of company package

File: src/main/resources/application-heroku.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+spring.jpa.hibernate.ddl-auto=create
```
Comment (404-bug-not-found) [inline src/main/resources/application-heroku.properties:1]: ddl-auto should be set to update

File: src/main/resources/application-local.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+spring.datasource.hikari.connectionTimeout=20000
+spring.datasource.hikari.maximumPoolSize=5
+
+## PostgreSQL
+spring.datasource.url=jdbc:postgresql://invoicify-postgres:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=open02
+
+#drop n create table again, good for testing, comment this in production
+spring.jpa.hibernate.ddl-auto=create
```
Comment (404-bug-not-found) [inline src/main/resources/application-local.properties:11]: Please set ddl-auto to update

**Review by 404-bug-not-found — CHANGES_REQUESTED** [review]
Please check review comments

**Pull request merged**